railties/lib/rails/generators/rails/app/templates/app/assets/config/manifest.js.tt

```diff
@@ -0,0 +1,8 @@
+
+<% unless options.api? -%>
+//= link_tree ../images
+<% end -%>
+<% unless options.skip_javascript -%>
+//= link_directory ../javascripts .js
```

Comment:
I think it is, because:
1. Apps with multiple asset bundles are typically set up this way.
2. Linking assets of the proper type (e.g. `.js` and `.css` not `.coffee`/`.es6` and `.scss`) with `link_directory` and `_tree` is not obvious to anyone. Defaulting to show the right way is instructive.
3. Precompiling multiple bundles and even all files will become more common as people migrate to HTTP/2.
3. Precompiling additional assets is usually cheap since the assets are already cached due to a `require_tree .` reference in `application.js/css`.